pkg/configapi/v1/types.go

```diff
@@ -0,0 +1,336 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
```

Comment:
Two things that block expapi to have v1alpha1 are:
1. a v1alpha1 codec won't understand Status. See [here](https://github.com/kubernetes/kubernetes/blob/master/pkg/apiserver/apiserver.go#L308). Kris has an WIP PR #13823 which will solve this problem.
2. you need to refactor the registered package [here](https://github.com/kubernetes/kubernetes/blob/master/pkg/api/latest/latest.go#L70), which I'm working on (#13833)

I think configapi is also affected by these them. Hopefully they will be fixed soon.